compiler/quilt/test/test_import.py

```diff
@@ -198,6 +198,130 @@ def test_multiple_package_dirs(self):
             from quilt.data.foo import multiple1
             assert multiple1.dataframes
 
+             # Check search order again just to make sure it doesn't return same named empty package
+            from quilt.data.foo import multiple1
+            assert multiple1.dataframes
```

Comment:
Python caches imports, so this can't fail because it was already imported above.

Reply:
@Kurlov  pls remove and add a comment ("no value in re-checking the search order because of python import caching")